test/test_helper.exs

```diff
@@ -4,6 +4,12 @@ Mix.shell(Mix.Shell.Process)
 System.put_env("ECTO_EDITOR", "")
 Logger.configure(level: :info)
 
+defmodule Ecto.Integration.AdapterTypes do
+  # Builtin uuid type is Ecto.UUID, many adapters will use it, but some may use
+  # it own implementation.
+  def uuid, do: Ecto.UUID
+end
```

Comment:
I don't think this is ever loaded for adapters, so we can probably get rid of these changes. :)